fm4tlp/tgbl_comment_dataprep.py

```diff
@@ -25,15 +25,15 @@
 
 import os
 import pickle
-
+import sys
 from absl import app
 from absl import flags
 import networkx as nx
 import numpy as np
 import pandas as pd
 import tensorflow.compat.v1 as tf
 import tqdm
-
+from tensorflow.compat.v1.io import *
```

Comment:
Please remove this import (we cannot ever accept module-wide imports i.e. with *). tf is already accessible via `import tensorflow.compat.v1 as tf` above. Call gfile with `tf.io.gfile` throughout. Thanks